runelite-client/src/main/java/net/runelite/client/plugins/grid/GridPlugin.java

```diff
@@ -23,44 +24,79 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.inventorygrid;
+package net.runelite.client.plugins.grid;
 
 import com.google.inject.Inject;
 import com.google.inject.Provides;
+import lombok.Getter;
+import net.runelite.api.Client;
+import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetID;
+import net.runelite.api.widgets.WidgetInfo;
 import net.runelite.client.config.ConfigManager;
+import net.runelite.client.eventbus.Subscribe;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 import net.runelite.client.ui.overlay.OverlayManager;
 
 @PluginDescriptor(
-	name = "Inventory Grid",
-	description = "Shows a grid over the inventory and a preview of where items will be dragged",
+	name = "Grid",
```

Comment:
fwiw I think this should still be named the "Inventory Grid" plugin, even if it has functionality to show a similar grid over the bank interface. Renaming plugins is rarely not a pain for users.